.github/workflows/eks-tunnel.yaml

```diff
@@ -117,6 +121,12 @@ jobs:
           python cilium-sysdump.zip --output cilium-sysdump-out
         shell: bash {0} # Disable default fail-fast behaviour so that all commands run independently
 
+      - name: Uninstall and make sure the 'aws-node' DaemonSet has been rescheduled
+        if: ${{ success() }}
+        run: |
+          cilium uninstall --wait
+          [[ $(kubectl -n kube-system get ds/aws-node -o jsonpath='{.status.currentNumberScheduled}') == 2 ]]
```

Comment:
I'm afraid that (i) this may be flaky as the `aws-node` pods may take variable amount of time to be scheduled and (ii) one day we may bump the number of nodes and forget this. Would it make sense for us to (i) wait until this equals (ii) the number of nodes?

Reply:
Replaced with a new version where we look for the annotation on the DaemonSet and check that is has been properly removed.